nds-org/workbench-helm-chart
# Re-use wildcard certificate across multiple ingress rules

## Problem
A PR was merged earlier today without a change that allows us to reuse the wildcard cert across different ingresses/namespaces

## Approach
* Remove the issuer/cluster issuer from the Workbench ingress rules
* Remove the `secretName` from the auth/open ingress rules
* Change the name of `ndslabs-tls-root` to `ndslabs-tls` (this is now reused throughout)

## How to Test
1. Checkout and run this branch locally
2. Populate `values.yaml` appropriately
3. Deploy the Helm chart
4. Execute `kubectl get cert`
    * You should see `ndslabs-tls` listed as a cert (it should match your configured `secretName`)
    * NOTE: If you ran the previous code, you may also see older versions of `ndslabs-tls-auth` and ndslabs-tls-open` listed
